packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -169,100 +186,122 @@ const App = (props) => {
             >
                 <CategoriesProvider categories={allCategories}>
                     <CurrencyProvider currency={currency}>
-                        <Seo>
-                            <meta name="theme-color" content={THEME_COLOR} />
-                            <meta name="apple-mobile-web-app-title" content={DEFAULT_SITE_TITLE} />
-                            <link
-                                rel="apple-touch-icon"
-                                href={getAssetUrl('static/img/global/apple-touch-icon.png')}
-                            />
-                            <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
-
-                            {/* Urls for all localized versions of this page (including current page)
+                        <SiteProvider site={site}>
+                            <LocaleProvider locale={locale}>
```

Comment:
I think LocaleProvider is somewhat redundant, as there's already `<IntlProvider>` from line 169 above.

Reply:
@vmarta The reason why we have a `useLocale` is because `intl` does not know about locale alias. While we might not use it in our implementation, we want to open that feature for the users.

Reply:
Ok got it. All right, let's have both. I think I picture them like this:
- `IntlProvider` for the translations
- `LocaleProvider` for multi sites and locales